lib/reek/cli/options.rb

```diff
@@ -9,7 +9,7 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 9 }
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 10 }
```

Comment:
But none of the instance variables below is used for memoization @mvz :)
In any case, we should group those options. And one of these days, I'm gonna look into that.